packages/appium/docs/en/developing/publishing-packages.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
typo in the file name `pulishing`

Reply:
oo..

Reply:
https://github.com/appium/appium/pull/21723/commits/4f338a0ab304c1eefff4a2812fb9a8dc9bbff72d